packages/front-end/pages/design-system/index.tsx

```diff
@@ -450,6 +452,52 @@ export default function DesignSystemPage() {
         />
       </div>
 
+      <div className="appbox p-3">
+        <h3>Select</h3>
+        <Flex direction="column" gap="3" maxWidth="300px">
+          <Select
+            label="Select"
+            defaultValue="carrot"
+            value={selectValue}
+            setValue={setSelectValue}
+          >
+            <SelectItem value="apple">Apple</SelectItem>
+            <SelectItem value="carrot">Carrot</SelectItem>
+            <SelectSeparator />
+            <SelectItem value="apple-pie" disabled>
```

Comment:
Do you maybe want to add the text  "(disabled)" at the end? I couldn't figure out why I could select it when I picked it in the second dropdown (just because they use the same setter function). It also doesn't do the cursor thing to symbolize disabled like it does when the full selector is disabled, but maybe that's a Radix issue.